core/modules/parser/SelectParser.cc

```diff
@@ -55,6 +56,17 @@
 #include "parser/SqlSQL2Lexer.hpp"
 #include "parser/SqlSQL2TokenTypes.hpp"
 #include "query/SelectStmt.h"
+#include "util/IterableFormatter.h"
+
+// these must be included before Log.h because they have a function called LOGS
+// that conflicts with the LOGS macro defined in Log.h
```

Comment:
It absolutely is scary. I didn't look very hard for a different solution, but I assumed there wasn't a fix other than changing code in the antlr4 package. We could brainstorm other options?

Reply:
If it's in ANTRL4 then I guess we don't seem to have any other options. Perhaps just avoiding using LSST Logger within this code? Unfortunately that may be impractical as you may still need to log some debugging info. Potentially this could be resolved via adding a error/debug reporting method in your class to forward messages to the Logger.
In any case any afford to bypass this problem may not be worth a result.